### PR TITLE
Release 5.1.0

### PR DESCRIPTION
**Pull request by zilchms**
includes: #151 that has the skip-changelog label (so it makes no difference for the release)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -122,6 +132,19 @@ These should not affect the functionality of the module.
...
+ * Change return format on dns_srv and dns_mx to an array of hashes. This
+   removes the need for the optional field parameter. That can be fetched using
+   the map function in Puppet 4.x+ instead.
+ * Use the Puppet 4 function API.
+
+2.0.1:
+ * correctly convert array of Resolv::IPv4 or Resolv::IPv6 to array strings
+
+2.0.0:
+ * Raise an exception on empty replies.
```
Comment (TheMeier) [inline CHANGELOG.md:147]: Where does this come from?

Comment (zilchms) [reply CHANGELOG.md:147]: automatically generated by the rake task, but i dont know why it either wasnt there previously or is there now

Comment (TheMeier) [reply CHANGELOG.md:147]: https://github.com/voxpupuli/puppet-dnsquery/blob/master/HISTORY.md

Comment (zilchms) [reply CHANGELOG.md:147]: ahaaaa, will do another PR to fix that and rebase this one then

File: HISTORY.md [hunk 2]
```diff
@@ -1,13 +0,0 @@
-3.0.0:
```
Comment (bastelfreak) [inline HISTORY.md:1]: usually we try to keep the files, but it looks like it wasn't used in the past either.

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**